app/views/posts/show.html.erb

```diff
@@ -6,14 +6,19 @@
       </div>
 
       <div class="mr-5">
-        <p>comments: <span class="font-bold"><%= @post.comments_counter%>, </span>Likes: <span class="font-bold"><%= @post.likes_counter %></span></p>
+        <p>comments: <span class="font-bold"><%= @post.comments_counter %>, </span>Likes: <span class="font-bold"><%= @post.likes_counter %></span></p>
       </div>
     </div>
 
     <div>
       <p class="mt-5 ml-3" ><%= @post.text %></p>
     </div>
+
+    <div class="mb-5 mt-5 ml-3 flex gap-5">
+      <a class="border-solid border-4 border-black cursor-pointer"><%= render partial: 'shared/add_like' %></a>
+      <a href="/users/<%= @user.id %>/posts/<%= @post.id %>/comments/new" class="border-4 border-solid border-black"> New comment</a>
+    </div>
   </div>
```

Comment:
- [ ] Good job on making some changes however we  now have a different error in the post controller.

Screenshot
![image](https://github.com/rmiyoyo/blog-app/assets/81932478/0bc07f11-1062-489e-9835-87d8f24edb99)

Videocast
[cinnamon-2023-12-23T235341+0200.webm](https://github.com/rmiyoyo/blog-app/assets/81932478/77d991e9-ea50-43a5-9a6b-3f5e1c0bcf21)

Reply:
@H

> * [ ]  Good job on making some changes however we  now have a different error in the post controller.
> 
> Screenshot ![image](https://private-user-images.githubusercontent.com/81932478/292646616-0bc07f11-1062-489e-9835-87d8f24edb99.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMzNzAyMDIsIm5iZiI6MTcwMzM2OTkwMiwicGF0aCI6Ii84MTkzMjQ3OC8yOTI2NDY2MTYtMGJjMDdmMTEtMTA2Mi00ODllLTk4MzUtODdkOGYyNGVkYjk5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjIzVDIyMTgyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc0MTQxNjY1NDExNTNkOGVkZmVlNWY2ODJlZDFhMWI3NTM4N2E3NzcxZDZkMDg5ODUwOTJmYWY0YWIyNjE1ZTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.DM5ToisJ54qKI_jLHA4sZo3I5mxdTZI_3EvAM9GZ2fk)
> 
> Videocast [cinnamon-2023-12-23T235341+0200.webm](https://github.com/rmiyoyo/blog-app/assets/81932478/77d991e9-ea50-43a5-9a6b-3f5e1c0bcf21)

Hi Munsa, I am out of reviews, do you have an idea how to solve the problem? I am currently on extra reviews

Reply:
Hey @rmiyoyo, I wish I could help but I see the time, slack me if you have time

Reply:
@Munsa1 I sent you a slack